test/new_relic/agent/distributed_tracing/trace_context_cross_agent_test.rb

```diff
@@ -253,7 +253,11 @@ def verify_attributes(test_case_attributes, actual_attributes, event_type)
               %Q(Unexpected #{event_type} attribute "#{key}")
           end
 
-          # TODO: check vendors in test_case_attributes
+          (test_case_attributes['vendors'] || []).each do |k, v|
```

Comment:
Would you please explain the need for `|| []`? I want to make sure the TODO is fully satisfied by there always being vendor checks performed.

Reply:
addressed with https://github.com/newrelic/newrelic-ruby-agent/pull/1692